Kai.BaseModule/KaiSDK.cs

```diff
@@ -528,11 +542,11 @@ private static void DecodeConnectedKais(JObject input)
 			{
 				var kai = token.ToObject<JObject>();
 				var kaiID = kai[Constants.KaiId].ToObject<int>();
-				var hand = kai[Constants.Hand].ToObject<string>();
+				var hand = kai[Constants.Hand]?.ToObject<string>();
```

Comment:
The SDK is supposed to send the hand of the Kai while connection. I don't think we should keep this as an optional parameter.

Reply:
Alright